bigchaindb_driver/utils.py

```diff
@@ -0,0 +1,41 @@
+from bigchaindb.common.transaction import Asset
+
+
+class CreateOperation:
+    pass
+
+
+class TransferOperation:
+    pass
+
+
+ops_map = {
+    'CREATE': CreateOperation,
+    'TRANSFER': TransferOperation,
+}
+
+
+def _normalize_operation(operation):
+    try:
+        operation = operation.upper()
+    except AttributeError:
+        pass
+
+    try:
+        operation = ops_map[operation]()
+    except KeyError:
+        pass
+
+    return operation
+
+
+def _normalize_asset(asset, is_transfer=False):
+    if is_transfer:
```

Comment:
Not sure I fully understand your questions. Here are a few points that may help to clarify:

1. The function is purely for internal use.
2. The function is to help isolate dependencies on `common`.
3. The function is meant to take a `dict` representing an asset, and convert it to a normalized form, which for the time being is an `Asset` instance.